src/NzbDrone.Core/ImportLists/Plex/List/PlexListSettings.cs

```diff
@@ -0,0 +1,22 @@
+using FluentValidation;
```

Comment:
I'm not actually sure I need this file at all as the base settings contain... everything